noxfiles/docs_nox.py

```diff
@@ -0,0 +1,65 @@
+"""Contains the nox sessions for developing docs."""
+import nox
+from constants_nox import CI_ARGS, RUN
+from docker_nox import build_local
+
+
+def docs_build(session: nox.Session) -> None:
```

Comment:
Session decorator is missing.

```diff
+ @nox.session()
def docs_build(session: nox.Session) -> None:
```

Reply:
did this on purpose since it is used by `docs_build_ci` and `docs_build_local`, but there is a better pattern for this for sure. I'll parametrize it